### PR TITLE
Enforce consistent style using flake8

### PR DESCRIPTION
**Pull request by jlaine**

### CODE REVIEW

**Comment by jlaine** [comment]
@mikeboers we had discussed this but I hadn't managed to set up flake8 properly for cython. I have drawn inspiration from what `uvloop` does in its `setup.cfg`. Let me know how you feel about this.

**Comment by jlaine** [comment]
Good call separating flake8 from the rest of the tests. If you're happy with the PR as is let's hit the big green button!

**Comment by mikeboers** [comment]
This has been heavily rebased/edited, and merged.

File: av/audio/resampler.pyx [hunk 1]
```diff
@@ -99,17 +99,17 @@ cdef class AudioResampler(object):
                 err_check(lib.av_opt_set_int(self.ptr, 'in_sample_rate',     self.template.ptr.sample_rate, 0))
                 err_check(lib.av_opt_set_int(self.ptr, 'out_sample_rate',    self.rate, 0))
                 err_check(lib.swr_init(self.ptr))
-            except:
+            except AVError:
```
[the remark this answers is not in the document]
Comment (jlaine) [reply av/audio/resampler.pyx]: It's not a dumb question, flake8 did indeed pick this up, rightly saying the bare excepts are dangerous.

File: av/option.pyx [hunk 2]
```diff
@@ -25,7 +25,7 @@ cdef EnumType _OptionType = define_enum('OptionType', (
     ('RATIONAL', lib.AV_OPT_TYPE_RATIONAL),
     ('BINARY', lib.AV_OPT_TYPE_BINARY),
     ('DICT', lib.AV_OPT_TYPE_DICT),
-    #('UINT64', lib.AV_OPT_TYPE_UINT64), # Added recently, and not yet used AFAICT.
+    # ('UINT64', lib.AV_OPT_TYPE_UINT64), # Added recently, and not yet used AFAICT.
     ('CONST', lib.AV_OPT_TYPE_CONST),
```
[the remark this answers is not in the document]
Comment (jlaine) [reply av/option.pyx]: Such code should in my opinion either actually get un-commented or removed altogether.. How recent was "recently"?

**Comment by jlaine** [comment]
Could you maybe delete your temporary branches once they get merged? It's getting pretty hard to track which branches still have unmerged changes!

**Comment by mikeboers** [comment]
I pulled out `flake8` and `stage/develop` at least. Do you have a handy way to identify redundant ones?

**Comment by mikeboers** [comment]
2 more.

**Comment by jlaine** [comment]
I wish I did.. but don't :(

A potential candidate: https://github.com/mikeboers/PyAV/commits/pyio-seek

**Pull request closed without merging**